modules/auth/ldap/ldap.go

```diff
@@ -278,12 +306,12 @@ func (ls *Source) SearchEntry(name, passwd string, directBind bool) *SearchResul
 
 	var isAttributeSSHPublicKeySet = len(strings.TrimSpace(ls.AttributeSSHPublicKey)) > 0
 
-	attribs := []string{ls.AttributeUsername, ls.AttributeName, ls.AttributeSurname, ls.AttributeMail}
+	attribs := []string{ls.AttributeUsername, ls.AttributeName, ls.AttributeSurname, ls.AttributeMail, ls.UserUID}
```

Comment:
If `ls.UserUID` is blank is this actually ok?

Reply:
Will defer to @lafriks as I have no idea. :sweat_smile:

Reply:
I think for belt and braces protection:

```suggestion
	attribs := []string{ls.AttributeUsername, ls.AttributeName, ls.AttributeSurname, ls.AttributeMail}
	if len(strings.TrimSpace(ls.UserUID)) > 0 {
		attribs = append(attribs, ls.UserUID)
	}
```